concrete/src/Api/OAuth/Controller.php

```diff
@@ -180,8 +181,11 @@ public function handleLogin(AuthorizationRequest $request)
             /** @var User $user */
             $user = $userRepository->findOneBy([$emailLogin ? 'uEmail' : 'uName' => $user]);
 
+            $app = Application::getFacadeApplication();
```

Comment:
Since this class is a controller, you should be able to use `$this->app` instead of having to write `$app = Application::getFacadeApplication();`

Reply:
Nope, I'm wrong, this class is not a standard *controller* class

Reply:
No probs, PHPStorm flagged that $this->app didn't exist so I went with fetching $app.